make/tools.mk

```diff
@@ -16,36 +17,38 @@
 ##############################
 
 # Set up ARM (STM32) SDK
-ARM_SDK_DIR ?= $(TOOLS_DIR)/gcc-arm-none-eabi-10.3-2021.10
+ARM_SDK_DIR ?= $(TOOLS_DIR)/arm-gnu-toolchain-12.2.mpacbti-rel1
```

Comment:
Overridden later, with os dependent path

Reply:
Removed.

Reply:
Yes removed but then can't find the tool. So have put it back.

Reply:
It's overridden by adding the OS specific path. Renamed to ARM_SDK_BASE_DIR